test/addons-napi/3_callbacks/binding.c

```diff
@@ -0,0 +1,49 @@
+#include <node_api.h>
+
+#define NAPI_CALL(theCall)                                                \
+  if (theCall != napi_ok) {                                               \
```

Comment:
Fixed in https://github.com/nodejs/node/pull/11975/commits/20b6cf24dda5f7c3be607b08021dc8a05a36c429